modules/administration/pages/backup-restore.adoc

```diff
@@ -473,8 +503,12 @@ rcpostgresql start
 ----
 spacewalk-service start
 ----
+////
 
 
+////
+// FIXME: does this still make sense?
```

Comment:
this section needs to be dropped or re-written. What we have in here is not true anymore.